### PR TITLE
Primer29

### PR DESCRIPTION
**Pull request by marby3**
## Description
This is firmware for a simple macropad keyboard using ProMicro.
I am an employee of the company that owns this vid.
I have official permission to use it from Yusha Kobo.
https://www.the-sz.com/products/usbid/index.php?v=0x3265&p=&n=

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/yushakobo/primer29/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# primer29
+
+![primer29](imgur.com image replace me!)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [yushakobo](https://github.com/marby3)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```
Comment (waffle87) [inline keyboards/yushakobo/primer29/readme.md:9]: Please fill out these fields.

File: keyboards/yushakobo/primer29/rules.mk [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# This file intentionally left blank
+CUSTOM_MATRIX = lite
+
```
Comment (waffle87) [inline keyboards/yushakobo/primer29/rules.mk:3]:
```suggestion
```

File: keyboards/yushakobo/primer29/info.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    "maintainer": "yushakobo",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
```
Comment (waffle87) [inline keyboards/yushakobo/primer29/info.json:13]:
```suggestion
    "matrix_pins": {
        "custom_lite": true,
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/yushakobo/primer29/readme.md (hunk 1 above)
Comment (marby3) [reply keyboards/yushakobo/primer29/readme.md]: Sorry, I fill out these fields!

File: keyboards/yushakobo/primer29/rules.mk (hunk 2 above)
Comment (marby3) [reply keyboards/yushakobo/primer29/rules.mk]: This change makes it impossible to build, so it is being returned.

File: keyboards/yushakobo/primer29/info.json (hunk 3 above)
Comment (marby3) [reply keyboards/yushakobo/primer29/info.json:13]: This change makes it impossible to build, so it is being returned.

File: keyboards/yushakobo/primer29/readme.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# primer29
+
+![primer29](https://i.imgur.com/EhQxZ4R.jpeg)
```
Comment (waffle87) [inline keyboards/yushakobo/primer29/readme.md:3]:
```suggestion
![primer29](https://i.imgur.com/EhQxZ4Rh.jpeg)
```

File: keyboards/yushakobo/primer29/readme.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+* Keyboard Maintainer: [yushakobo](https://github.com/yushakobo)
+* Hardware Supported: primer29 PCB, Pro Micro (ATmega32U4)
+* Hardware Availability: [yushakobo Shop](https://yushakobo.jp/shop/primer29/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make yushakobo/primer29:default
+    or
+    qmk compile -kb yushakobo/primer29 -km default
```
Comment (waffle87) [inline keyboards/yushakobo/primer29/readme.md:15]:
```suggestion
```

File: keyboards/yushakobo/primer29/readme.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+Flashing example for this keyboard:
+
+    make yushakobo/primer29:default
+    or
+    qmk flash -kb yushakobo/primer29 -km default
```
Comment (waffle87) [inline keyboards/yushakobo/primer29/readme.md:21]:
```suggestion
```

File: keyboards/yushakobo/primer29/matrix.c
Comment (drashna) [inline keyboards/yushakobo/primer29/matrix.c:1]: Could you run a format pass? 
`qmk format-c`, or the like.

**Review by drashna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**